images/win/scripts/Tests/Browsers.Tests.ps1

```diff
@@ -0,0 +1,160 @@
+Describe "Chrome" {
+    Context "WebDriver" {
+        It "ChromeWebDriver environment variable exists" {
+            $env:ChromeWebDriver | Should -Not -BeNullOrEmpty
+        }
+
+        It "ChromeWebDriver should be equal C:\SeleniumWebDrivers\ChromeDriver" {

```

Comment:
Do we really need to check the path? I believe it is not important the accurate path, just need to make sure that environment variable points correctly.

Reply:
Anyway, I suggest to merge all 3 checks
- ChromeWebDriver environment variable exists
- ChromeWebDriver should be equal C:\SeleniumWebDrivers\ChromeDriver
- ChromeWebDriver path exists

to the single `It`

Reply:
The same for other browsers

Reply:
Done.